### PR TITLE
[chore] bump collector deps to v0.83.0

### PR DESCRIPTION
**Pull request by codeboten**

### CODE REVIEW

File: collector/receiver/telemetryapireceiver/go.mod [hunk 1]
```diff
@@ -6,14 +6,14 @@ replace github.com/open-telemetry/opentelemetry-lambda/collector => ../../
 
 require (
 	github.com/golang-collections/go-datastructures v0.0.0-20150211160725-59788d5eb259
-	github.com/open-telemetry/opentelemetry-lambda/collector v0.81.0
+	github.com/open-telemetry/opentelemetry-lambda/collector v0.83.0
```
Comment (Aneurysm9) [inline collector/receiver/telemetryapireceiver/go.mod]: Should this be bumped later when a tag exists for this module at this version?

Comment (codeboten) [reply collector/receiver/telemetryapireceiver/go.mod]: This is overwritten by a replace anyways, but i don't feel strongly one way or the other.

This question makes me realize that we don't currently publish any tags for this package :(

**Review by tylerbenson — APPROVED** [review]

File: collector/receiver/telemetryapireceiver/go.mod (hunk 1 above)
Comment (Aneurysm9) [reply collector/receiver/telemetryapireceiver/go.mod]: Can we leave it unchanged for now, then, and bump it at the next release when tags are created?

Comment (codeboten) [reply collector/receiver/telemetryapireceiver/go.mod]: Done, i'll create a ticket to ensure module tags are published

**Review by Aneurysm9 — APPROVED** [review]

**Pull request merged**